### PR TITLE
fix twitter for light theme

### PR DESCRIPTION
**Pull request by erik-sn**
Removes coded background and text colors and falls back to theme. Minor styling updates.

image

### CODE REVIEW

File: phone/src/ui/components/Modal.tsx [hunk 1]
```diff
@@ -5,7 +5,6 @@ import { Paper } from '@material-ui/core';
 const useStyles = makeStyles(() => ({
```
Comment (TasoOneAsia) [inline phone/src/ui/components/Modal.tsx:5]: Should be able to just pass an object without using an anon function when the `ThemeObject` does not need to be passed.
```ts
makeStyles({})
```

File: phone/src/apps/twitter/components/tweet/TweetMessage.tsx [hunk 2]
```diff
@@ -4,17 +4,15 @@ import { makeStyles } from '@material-ui/core/styles';
 import { TextField } from '@material-ui/core';
 import { usePhone } from '../../../../os/phone/hooks/usePhone';
 
-const useStyles = makeStyles((theme) => ({
+const useStyles = makeStyles(() => ({
```
Comment (TasoOneAsia) [inline phone/src/apps/twitter/components/tweet/TweetMessage.tsx:7]: Same as above
```ts
makeStyles({})
```

**Review by TasoOneAsia — CHANGES_REQUESTED** [review]
Aside from small changes for cleaning up `makeStyles` when not passing `ThemeObject`, LGTM 👍

**Comment by RockySouthpaw** [comment]
Looks fine to me as well. 

https://i.imgur.com/cbCcSch.png

**Review by RockySouthpaw — APPROVED** [review]

**Comment by erik-sn** [comment]
@TasoOneAsia good call, updated!

**Pull request merged**